sqlglot/dialects/presto.py

```diff
@@ -118,6 +119,12 @@ class Presto(Dialect):
     time_format = "'%Y-%m-%d %H:%i:%S'"
     time_mapping = MySQL.time_mapping
 
+    class Tokenizer(Tokenizer):
+        KEYWORDS = {
+            **Tokenizer.KEYWORDS,
+            "ROW": TokenType.STRUCT,
```

Comment:
We had talked about how this would probably not be a good idea. It seems like it works out in practice since the only time I ever see row is with `CURRENT ROW` and `CURRENT ROW` get parsed into the `CURRENT_ROW` token. So I think this should end up working out.

Reply:
right